README.md

```diff
@@ -1,3 +1,5 @@
 # Reconfigure
 
 A reconfiguration service for CoreOS applications.
+
+`$ bash launch 4` starts a cluster of 4.
```

Comment:
Use three as an example instead of four. Four is a weird number for a consensus.